README.md

```diff
@@ -96,16 +96,55 @@ If you're not using OpenWhisk in Bluemix or if you want to set up your Cloudant
 
 ## Listening for changes to a Cloudant database
 
+### Filter database change events
+
+You can defined a filter function, to avoid having unnecessary change events firing your trigger.
+
+To create a new filter function you can use an action.
+
+Create a json document file `design_doc.json` with the following filter function
+```json
+{
+  "doc": {
+    "_id": "_design/mailbox",
+    "filters": {
+      "by_status": "function(doc, req){if (doc.status != req.query.status){return false;} return true;}"
+    }
+  }
+}
+```
+
+Create a new design document on the database with the filter function
+
+```
+wsk action invoke /_/myCloudant/write -p dbname testdb -p overwrite true -P design_doc.json -r
+```
+```json
+{
+    "id": "_design/mailbox",
+    "ok": true,
+    "rev": "1-5c361ed5141bc7856d4d7c24e4daddfd"
```

Comment:
Oh this is just the output, not the input doc. the input doc is in design_doc.json.